JSQMessagesDemo/he.lproj/Localizable.strings

```diff
@@ -21,6 +21,7 @@
 "Send photo" = "שלח תמונה";
 "Send location" = "שלח מיקום";
 "Send video" = "שלח וידאו";
+"Send video thumbnail" = "Send video with thumbnail";
```

Comment:
It's just a placeholder for something who can translate that? Or just Google it? lol